templates/partials/applicantInfoModal.phtml

```diff
@@ -69,6 +69,11 @@
                         <p class="detail">Portfolio Url: <span id="portfolioUrl"></span></p>
                         <p class="detail">Plesk Hosting Url: <span id="pleskHostingUrl"></span></p>
                         <p class="detail">Github Education Link: <span id="githubEducationLink"></span></p>
+                        <p class="detail">Profile Link: <a id="userProfileLink" href="userProfileLink"></a> <button class="clipboard" onclick="copyToClipboard('#userProfileLink')"> <i class="glyphicon glyphicon-copy"></i>
```

Comment:
Can you make the link open in a new tab please

Reply:
Also inline event listeners are the devils work - please remove the onclick here and add an event listener in the JS instead